docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-add-endpoint.md

```diff
@@ -39,3 +53,23 @@ To use ybm CLI, you need to do the following:
 
 - Create an API key. Refer to [API keys](../../../managed-automation/managed-apikeys/).
 - Install and configure ybm CLI. Refer to [Install and configure](../../../managed-automation/managed-cli/managed-cli-overview/).
+
+Note that, unlike VPC peering, when connected to an application VPC using a private link, you do not need to add an IP allow list to your cluster.
+
+## Get started
+
+{{< sections/2-boxes >}}
+  {{< sections/bottom-image-box
+    title="Set up PrivateLink on AWS"
```

Comment:
Either change to "Set up AWS PrivateLink" or "Set up private link on AWS". I get it that AWS and Azure conventions are different , but in our docs , the left nav heading is different and then the tile heading is different and it sorts of seems confusing.